### PR TITLE
added a few basic cluster methods to use in.. stuff

### PR DESCRIPTION
**Pull request by rdallman**

### CODE REVIEW

File: worker/methods.go [hunk 1]
```diff
@@ -504,3 +504,31 @@ func EncryptPayloads(publicKey []byte, in ...Task) ([]Task, error) {
 	}
 	return tasks, nil
 }
+
+type Cluster struct {
```
Comment (nikhilm) [inline worker/methods.go]: All these seem like 'admin' like things, but end up being exposed publicly in the package. Just add doc comments to these saying they are subject to change without warning?

**Comment by nikhilm** [comment]
lgtm

File: worker/methods.go (hunk 1 above)
Comment (rdallman) [inline worker/methods.go]: these are actually the non admin things we advertise as part of the api

**Comment by rdallman** [comment]
thx

File: worker/methods.go (hunk 1 above)
Comment (nikhilm) [inline worker/methods.go]: Hmm... don't see them http://dev.iron.io/worker/reference/api/

Comment (rdallman) [inline worker/methods.go]: http://dev.iron.io/worker/hybrid/README.md go to clusters page

**Pull request merged**